NEWS.md

```diff
@@ -4,6 +4,11 @@ ClimaAtmos.jl Release Notes
 Main
 -------
 
+v0.27.4
+-------
+- Add artifact support for GCM-driven single column
+  PR [#3256](https://github.com/CliMA/ClimaAtmos.jl/pull/3256)
```

Comment:
Let's make this entry more informative. 

This PR brings a new capability to ClimaAtmos, namely, being able to decode `artifact` from YAML. Let's inform readers of this.

Reply:
Agreed!